packages/reassure-cli/src/commands/measure.ts

```diff
@@ -85,6 +100,16 @@ export const command: CommandModule<{}, MeasureOptions> = {
         type: 'boolean',
         default: true,
         describe: 'Outputs performance comparison results',
+      })
+      .option('branch', {
+        type: 'string',
+        default: '--not-specified--',
```

Comment:
I think that there should be no default value atm. In the subsequent PR we will try to get it from `git`.
```suggestion
```